ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -195,6 +224,34 @@ extension SignalProtocol {
 	}
 }
 
+extension SignalProtocol where Self.Value: OptionalProtocol {
+	/// Create a `RACSignal` that will observe the given signal.
+	///
+	/// - note: Any `interrupted` events will be silently discarded.
+	///
+	/// - returns: `RACSignal` instantiated from `self`.
+	public func toRACSignal() -> RACSignal {
```

Comment:
@NachoSoto your comment made me realize that the ActionProtocol.toRACCommand() overloads I originally had submitted with this PR aren't needed. Thus, I've removed them in b0fdc4c.